extensions/amp-intaker-chat-widget/0.1/test/validator-amp-intaker-chat-widget.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+    <title>amp-intaker-chat-widget example</title>
+    <link rel="canonical" href="amps.html">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-intaker-chat-widget"
+            src="https://cdn.ampproject.org/v0/amp-intaker-chat-widget-0.1.js"></script>
+  </head>
+  <body>
+
+    <amp-intaker-chat-widget layout="nodisplay"
```

Comment:
nit: could you a line above this that mentions that it is valid e.g. `<!-- Valid -->`

Reply:
https://github.com/ampproject/amphtml/pull/24337/commits/3571cb01d6f45e190ce3f2260219c833dc704818